tests/regression_tests/external_moab/test.py

```diff
@@ -0,0 +1,281 @@
+    clad_max_y = openmc.YPlane(6.0, name="maximum y")
+
+    clad_min_z = openmc.ZPlane(-6.0, name="minimum z")
+    clad_max_z = openmc.ZPlane(6.0, name="maximum z")
+
+    clad_cell = openmc.Cell(name="clad")
+    clad_cell.region = (-fuel_min_x | +fuel_max_x |
+                        -fuel_min_y | +fuel_max_y |
+                        -fuel_min_z | +fuel_max_z) & \
+                        (+clad_min_x & -clad_max_x &
+                         +clad_min_y & -clad_max_y &
+                         +clad_min_z & -clad_max_z)
+    clad_cell.fill = zirc_mat
+
+    bounds = (10, 10, 10)
+
+    water_min_x = openmc.XPlane(x0=-bounds[0],
+                                name="minimum x",
+                                boundary_type='vacuum')
+    water_max_x = openmc.XPlane(x0=bounds[0],
+                                name="maximum x",
+                                boundary_type='vacuum')
+
+    water_min_y = openmc.YPlane(y0=-bounds[1],
+                                name="minimum y",
+                                boundary_type='vacuum')
+    water_max_y = openmc.YPlane(y0=bounds[1],
+                                name="maximum y",
+                                boundary_type='vacuum')
+
+    water_min_z = openmc.ZPlane(z0=-bounds[2],
+                                name="minimum z",
+                                boundary_type='vacuum')
+    water_max_z = openmc.ZPlane(z0=bounds[2],
+                                name="maximum z",
+                                boundary_type='vacuum')
+
+    water_cell = openmc.Cell(name="water")
+    water_cell.region = (-clad_min_x | +clad_max_x |
+                         -clad_min_y | +clad_max_y |
+                         -clad_min_z | +clad_max_z) & \
+                         (+water_min_x & -water_max_x &
+                          +water_min_y & -water_max_y &
+                          +water_min_z & -water_max_z)
+    water_cell.fill = water_mat
+
+    # create a containing universe
+    geometry = openmc.Geometry([fuel_cell, clad_cell, water_cell])
+
+    ### Tallies ###
+
+    # Meshes
+    mesh_filename = "test_mesh_tets.h5m"
+
+    # Create a normal unstructured mesh to compare to
+    uscd_mesh = openmc.UnstructuredMesh(mesh_filename,'moab')
+
+    # Create filters
+    uscd_filter = openmc.MeshFilter(mesh=uscd_mesh)
+
+    # Create tallies
+    tallies = openmc.Tallies()
+    uscd_tally = openmc.Tally(name="unstructured mesh tally")
+    uscd_tally.filters = [uscd_filter]
+    uscd_tally.scores = ['flux']
+    uscd_tally.estimator = 'tracklength'
+    tallies.append(uscd_tally)
+
+    ### Settings ###
+    settings = openmc.Settings()
+    settings.run_mode = 'fixed source'
+    settings.particles = 100
+    settings.batches = 10
+
+    # Source setup
+    r = openmc.stats.Uniform(a=0.0, b=0.0)
+    theta = openmc.stats.Discrete(x=[0.0], p=[1.0])
+    phi = openmc.stats.Discrete(x=[0.0], p=[1.0])
+
+    space = openmc.stats.SphericalIndependent(r, theta, phi)
```

Comment:
Since this radius is always zero, this is the same as a point source:
```suggestion
    space = openmc.stats.Point()
```

Reply:
Looks like there are a few updates I can make to the unstructured mesh tests following this PR...